spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/RdbmsUriRegistry.java

```diff
@@ -81,6 +82,8 @@ public URI find(String name) {
 
 	@Override
 	public void register(String name, URI uri) {
+		Assert.notNull(uri, "Error when registering " + name + ": uri is required");
+		Assert.hasText(uri.toString(), "Error when registering " + name + ": uri must not be blank");
```

Comment:
Given the usage of `ResourceLoader` when loading from the URI, can we just validate if the URI string has a valid URI Scheme (at least starting with a string with colon)?